### DISCUSSION TITLE
How to @all or @someone?

### DISCUSSION THREAD

**Discussion by Miopha** [Terrance/SkPy#274] [Q&A] [answered]
Hi all,
I am looking for a method to @all or @someone in chats
I tried `ch.sendMsg('<at id="*">all</at>')` (that's what `ch.getMsgs()` got when I @all), but it only send plain text to chat.

**Comment by Terrance** [comment, marked as answer]
> `ch.sendMsg('<at id="*">all</at>')`

This will send your markup in plain text.  Did you mean `ch.sendMsg(..., rich=True)`?

See also: #153
